lib/lsperfm/core/stats.rb

```diff
@@ -0,0 +1,64 @@
+# encoding: utf-8
+require "thread"
+
+Thread.abort_on_exception = true
+
+module LogStash::PerfM
```

Comment:
Welcome to the team, naming is a hard problem. I wanted to actually give this module a name related with performance and meter, measure, etc... so I put PerfM, so the namespace was not big, big.

I would avoid Benchmark and Test as they use to be misleading for the expectations of people, I aim this module to do moar.